modules/registry-viewing-contents.adoc

```diff
@@ -42,7 +42,7 @@ the container image registry:
 . List the registry contents using the `oc rsh` command:
 +
 ----
-# oc rsh dc/podman-registry find /registry
+# oc rsh podman-registry find /registry
```

Comment:
In 4.0, only S3 storage is supported, there will be no content under registry folder. Plus, no need to ssh into node to access registry pod, user can use kube:admin account to login and then check the pods under project openshift-image-registry like below:
$ oc get pods
NAME                                               READY   STATUS    RESTARTS   AGE
cluster-image-registry-operator-764bd7f846-qqtpb   1/1     Running   0          78m
image-registry-79fb4469f6-llrln                    1/1     Running   0          77m
node-ca-hjksc                                      1/1     Running   0          73m
node-ca-tftj6                                      1/1     Running   0          77m
node-ca-wb6ht                                      1/1     Running   0          77m
node-ca-zvt9q                                      1/1     Running   0          74m